spec/jasmine.tests.js

```diff
@@ -39,7 +39,7 @@ module.exports = [
 
                 var $ = cheerio.load(data)
 
-                expect($('svg#inline-me').length).toBe(1)
+                expect($('svg#replace-me').length).toBe(1)
```

Comment:
All the changes on this file are because tests weren't appropriately updated on https://github.com/theGC/html-webpack-inline-svg-plugin/pull/21 since this PR adds the feature of copying all attributes from the `<img/>` to the `<svg/>`. And also because of the aforementioned reducer bug.